R/format_mr_results2.R

```diff
@@ -341,14 +339,15 @@ power_prune <- function(dat,method=1,dist.outcome="binary")
 				r2sum<-sum(r2) # sum of the r-squares for each SNP in the instrument
 				# F<-r2sum*(n-1-k)/((1-r2sum*k )
 				if(dist.outcome == "continuous"){
-					iv.se<- 1/sqrt(unique(dat2$samplesize.outcome)*r2sum) #standard error of the IV should be proportional to this
+					iv.se<- 1/sqrt(mean(dat2$samplesize.outcome)*r2sum) #standard error of the IV should be proportional to this
```

Comment:
It would feel a bit safer to me if you could you add in `na.rm = TRUE` to the `mean()` call, i.e.,

```r
iv.se<- 1/sqrt(mean(dat2$samplesize.outcome, na.rm = TRUE)*r2sum)
```